src/styles/_catalog.scss

```
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```

```suggestion
  grid-template-columns: repeat(4, 1fr);
```

this is a more convenient way 😉

---

Fixed, tnx!